src/locutus/model/terminology.py

```diff
@@ -67,14 +68,17 @@ def build_code(self, data, **kwargs):
             # pdb.set_trace()
             return Coding(**data)
 
-    def to_dict(self):
+    def to_dict(self, valid=None):
         obj = {"code": self.code, "display": self.display}
 
         if self.description != "":
             obj["description"] = self.description
 
         if self.system is not None:
             obj["system"] = self.system
+                    
+        if valid is not None:
+            obj["valid"] = valid

```

Comment:
If we are operating under the assumption that all codes should have a valid flag, but some were originally created before it existed, why not just treat valid=None as a True?